samples/matter/window_covering/sample.yaml

```diff
@@ -0,0 +1,35 @@
+sample:
+  name: Matter Window Covering
+  description: Matter Window Covering example
+tests:
+  matter.lock.debug:
+    build_only: true
+    platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp
+    integration_platforms:
+      - nrf52840dk_nrf52840
+      - nrf5340dk_nrf5340_cpuapp
+    tags: ci_build
+  matter.lock.release:
+    build_only: true
+    platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp
+    integration_platforms:
+      - nrf52840dk_nrf52840
+      - nrf5340dk_nrf5340_cpuapp
+    tags: ci_build
+    extra_args: CONF_FILE=prj_release.conf
+  matter.lock.no_dfu:
+    build_only: true
+    platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp
+    integration_platforms:
+      - nrf52840dk_nrf52840
+      - nrf5340dk_nrf5340_cpuapp
+    tags: ci_build
+    extra_args: CONF_FILE=prj_no_dfu.conf
+  matter.lock.low_power:
+    build_only: true
+    platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp
+    integration_platforms:
+      - nrf52840dk_nrf52840
+      - nrf5340dk_nrf5340_cpuapp
+    tags: ci_build
+    extra_args: "OVERLAY_CONFIG=../../overlay-low_power.conf"
```

Comment:
overlay-low_power.conf is located in the window covering sample directory, so you don't need to add `../../`

Reply:
Done.